config/initializers/job_configurations.rb

```diff
@@ -58,8 +58,9 @@
         cron: cron_24h,
         args: -> { [Time.zone.today] },
       },
+      # Idv Legacy Conversion Supplement Report to S3
       agreement_summary_report: {
-        class: 'Reports::AgreementSummaryReport',
```

Comment:
we still want to keep this old report, can we add a new entry instead?

Reply:
Ha, complete accident. Thanks